workspace sir vivor/commands.js

```diff
@@ -2037,6 +2037,20 @@ exports.commands = {
 		}
 		dd.updateModlog(user.name + " did .rmparts " + target);
 	},
+	
+	ddlog: function (target, user, room) {
+		if (!user.hasRank('survivor', '%')) return;
+		if (!("data" in dd.modlog)) return;
+		let buffer = '';
+		for (let i = 0; i < dd.modlog.data.length; i++) {
+			buffer += dd.modlog.data[i] + '\n';
+		}
+		Tools.uploadToHastebin(buffer, (success, link) => {
+			if (success) user.say(link);
```

Comment:
idk if i'll get an error here referencing an argument in a callback, hopefully not